packages/neuron-ui/src/components/OfflineSign/sign-dialog.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { useState, useRef, useCallback, useMemo, useEffect } from 'react'
```

Comment:
Please follow the filename convention: https://github.com/airbnb/javascript/tree/master/react#naming